File: CiscoPCCE.Toolkit/Bean/CallServer.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlRoot("server")]
    public class CallServer : BaseApiBean
    {
        [XmlElement("address")]
        public string? Address { get; set; }

        [XmlElement("messageBusNumber")]
        public int? MessageBusNumber { get; set; }

        [XmlElement("port")]
        public int? Port { get; set; }

        [XmlElement("subsystemNumber")]
        public int? SubsystemNumber { get; set; }
    }
}

File: JavaSrc/toolkit/bean/CallServer.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="server")
public class CallServer extends BaseApiBean {
  private String address;
  private Integer messageBusNumber;
  private Integer port;
  private Integer subsystemNumber;

  public String getAddress() {
     return this.address;
  }

  public void setAddress(String address) {
     this.address = address;
  }

  public Integer getMessageBusNumber() {
     return this.messageBusNumber;
  }

  public void setMessageBusNumber(Integer messageBusNumber) {
     this.messageBusNumber = messageBusNumber;
  }

  public Integer getPort() {
     return this.port;
  }

  public void setPort(Integer port) {
     this.port = port;
  }

  public Integer getSubsystemNumber() {
     return this.subsystemNumber;
  }

  public void setSubsystemNumber(Integer subsystemNumber) {
     this.subsystemNumber = subsystemNumber;
  }


}
